client/src/Movies/MovieList.js

```diff
@@ -34,6 +37,7 @@ export default class MovieList extends Component {
 function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
+    <Link to={`movie/$/movie.id`}>
```

Comment:
consider changing this line to look like this: ```<Link to={`/movies/${movie.id}`}>``` this way you are properly utilizing string literals, and at the same time you are reaching the appropriate endpoint ```/movies/:id```